src/core/api/index.js

```diff
@@ -170,3 +170,13 @@ export function categories({ api }) {
     state: api,
   });
 }
+
+export function logOutFromServer({ api }) {
+  return callApi({
+    auth: true,
+    credentials: true,
+    endpoint: 'accounts/session',
+    method: 'delete',
+    state: api,
+  });
```

Comment:
If it's a `200` I think it should be okay. I just put `{ "ok": true }` in the response because it seemed weird to return nothing.